config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml

```diff
@@ -367,9 +367,10 @@ presubmits:
         - --image-config-file=/go/src/k8s.io/test-infra/jobs/e2e_node/containerd/containerd-master/image-config.yaml
         - --image-config-dir=/home/prow
   - name: pull-kubernetes-node-e2e-alpha
-    cluster: k8s-infra-prow-build
```

Comment:
is community owned cluster also reason for suspected failure?

Reply:
replicating the job "pull-kubernetes-node-e2e". I'm super puzzled by it. Maybe try just quotes first

Reply:
wdyt?

Reply:
agreed lets try just quotes first